src/components/Navigation.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 
+//might want to adjust these styles or take them out and replace with tailwind classes..
```

Comment:
+1 on leaving notes like this, so it's obvious that it's pre-tailwind and possible todo. Recommendation: start comment with TODO: since many IDE editors know to look for those, and it's more obvious something ideally should happen here.